oauth2client/contrib/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
```

Comment:
Sadly there's not an example in the style guide to which I can refer, but the format of module doc strings should be
```
"""A single summary line on the same line as the opening triple-quote.

Zero or more additional paragraphs individually separated by single blank
lines.

The closing triple-quote goes at the end of the summary line if there
are zero additional paragraphs and goes on its own line if there are one or
more additional paragraphs.
"""
```
.